scripts/container_ops.sh

```diff
@@ -64,39 +71,73 @@ run_cmd() {
 ###################################################################
 
 # >>>>>>>  Initialization
+# Input vars
+REGISTRY_LOGIN=${1:-""}
+PUSH_TO_REGISTRY=${2:-""}
 
+# Static vars
+# TODO(improve logic for finding app.conf): Currently we have locked config location in infra/app.conf, we need to expand this feature
+# in order if someone wants to build something in folder_name/ and it has different registry from infra/app.conf.
 APP_CONFIG_FILE="${GITHUB_WORKSPACE}/infra/app.conf"
-if [[ -f "${APP_CONFIG_FILE}" ]]; then
-  echo "Application Config detected! Loading parameters..."
-  # shellcheck source=${GITHUB_WORKSPACE}/infra/app.conf
-  source "${APP_CONFIG_FILE}"
-fi
-
-if [[ -z "${CONTAINER_REPO}" ]]; then
-  echo "Docker root repository is not defined. Aborting docker operations."
-  exit 1
-fi
-
-CONTAINER_IMG_TAG="${CONTAINER_REPO}:${APP_VERSION}"
-
-# >>>>>>>  Start
-
-# as first, build docker image
-run_cmd build "${1}"
-
-# as second, try to login
-if [[ "${3}" == "true" ]]; then
-  run_cmd login
-fi
-# as third, publish the image if not specified otherwise
-# Parameter ${2} is `push_image` { true | false }
-if [[ "${2}" == "true" ]]; then
-  run_cmd push
-else
-  echo "(push = ${2}) Docker push disabled by user, image will not pushed to the Registry"
-fi
-
-# finally, cleanup
-run_cmd cleanup
-
-# >>>>>>>  End
+CONTAINER_DIR="infra/"
+
+
+main(){
+ 
+  # Check if app.conf is available if not exit the script 
+  if [[ -f "${APP_CONFIG_FILE}" ]]; then
+    echo "Application Config detected! Loading parameters..."
+    # shellcheck source=${GITHUB_WORKSPACE}/infra/app.conf
+    source "${APP_CONFIG_FILE}"
+  
+  else
+
+    echo ""
+    echo "Application config is not available in ${APP_CONFIG_FILE}."
+    echo ">>> STOPPING! <<<"
+    echo ">>> STOPPING! <<<"
+    echo ""
+    exit 1
```

Comment:
APP_CONFIG_FILE is not a mandatory parameter. If doesn't exist, just print a message about that. 

CONTAINER_REPO is mandatory because we can't build container without repo, but even in that case we can set CONTAINER_REPO to  `localreg` for instance.